db/migrate/20160502145559_remove_technology_id_field_from_importable_attributes.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveTechnologyIdFieldFromImportableAttributes < ActiveRecord::Migration
+  def change
+    remove_index :importable_attributes, :technology_id_and_name
+    remove_column :importable_attributes, :technology_id
+    add_index :importable_attributes, [:technology_key, :name], unique: true
```

Comment:
This would be a good opportunity to add a validation to ImportableAttribute to catch duplicate indices before attempting to save.

```ruby
validates :name, uniqueness: { scope: :technology_key, case_sensitive: false }
```